src/pcre2_compile.c

```diff
@@ -10206,13 +10206,36 @@ if (length > MAX_PATTERN_SIZE)
   goto HAD_CB_ERROR;
   }
```

Comment:
This block here could be in an "#else" block for the code you added.

Sorry, really minor nit.

Reply:
I am not sure I follow. I wanted to be in the safe side here, since `MAX_PATTERN_SIZE - length >= 0` after the check and never underflows.